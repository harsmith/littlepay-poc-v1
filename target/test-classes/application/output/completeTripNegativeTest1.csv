Started,Finished,DurationSecs,FromStopId,ToStopId,ChargeAmount,companyId,BusID,PAN,Status
22-01-2018 13:15:00,22-01-2018 13:15:00,0,Stop1,Stop1,$7.30,Company10,Bus37,3528000700000000,INCOMPLETE
