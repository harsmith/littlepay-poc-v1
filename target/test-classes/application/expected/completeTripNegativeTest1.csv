Started,Finished,DurationSecs,FromStopId,ToStopId,ChargeAmount,companyId,BusID,PAN,Status
22-01-2018 13:15:00,22-01-2018 13:15:00,300,Stop1,Stop1,$3.25,Company10,Bus37,3528000700000000,COMPLETE
